data-model/SDP-data-model-templates/person_sdp-template.json

```diff
@@ -1,8 +1,15 @@
 {
-  "id": "ARK/URI(?)",
+  "id": "URI",
   "label": "",
   "type": "CV",
   "uniform_name": "",
+  "ns_name": "",
+  "alt_name": [
+    "array of strings"
+    ],
+  "role": [
```

Comment:
Adding some thoughts here so I don't forget when we discuss...I'm assuming this field will be populated as someone adds references to a person record. For example, if they mark an attributed author in a textual artifact, 'attributed_author' will appear in this role list. (So this wouldn't get populated by hand).

Would it be worth also associating below (in l. 79-81) the related/associated ms/object with the nature of that relation? So we would, instead of a list of IDs, have an array of objects with the ID and the role/relation the person has with that object? This wouldn't be organized by role type, and it also would have a lot of repeats, so it could still be worth it to have a list of distinct roles/relations attested in our data.

Reply:
role field will be an array of objects that associate the role label/value with the object IDs for the objects that attest that particular role:

```
[{
    label: "author",
    assoc_objs: ["IDs"]
}]